app/javascript/components/reverted.js

```diff
@@ -1,5 +1,6 @@
 import template from './reverted.html'
 
 export default template({
-  data () { return {} }
+  data () { return {} },
+  props: ['statement', 'page', 'country']
```

Comment:
Would be nice if we could clear up this repetition somehow?

Reply:
Yeah it would. Having read through the Vue docs a bit more I don't think there is a built in way of handling component inheritance. For now I don't think its really worth the time investigating this.